Source/WebCore/css/CSSFunctionValue.cpp

```diff
@@ -26,18 +26,11 @@
 #include "config.h"
 #include "CSSFunctionValue.h"
 
-#include <wtf/text/StringBuilder.h>
-
 namespace WebCore {
     
-String CSSFunctionValue::customCSSText(Document* document) const
+String CSSFunctionValue::customCSSText() const
```

Comment:
Given the recursive nature, I kinda think implementing this using a StringBuilder passed in would likely be more efficient (though, that would probably require a separate function that took the StringBuilder since this this one has to return a String).

Reply:
I agree that we ought to find a way to build CSS serialization that uses a `StringBuilder` rather than intermediate strings for better performance.

Reply:
But I won’t make that change here.